robot_calibration/src/led_finder.cpp

```diff
@@ -247,9 +249,40 @@ bool LedFinder::find(robot_calibration_msgs::CalibrationData * msg)
   sensor_msgs::PointCloud2Iterator<float> iter_cloud(cloud, "x");
 
   // Export results
-  msg->observations.resize(2);
-  msg->observations[0].sensor_name = camera_sensor_name_;
-  msg->observations[1].sensor_name = chain_sensor_name_;
+  // Handle multiple finders
+  int idx_cam = -1;
+  int idx_chain = -1;
+
+  if (msg->observations.size() == 0)
```

Comment:
we probably don't need this block -- each finder should be the exclusive creator for each sensor, right? thus, we should always just append two new observations